content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,261 @@ weight: 2
+
+* [Introduction](#introduction)
+* [Deciding if you need A Waypoint Proxy](#deciding-if-you-need-a-waypoint-proxy)
+* [Current Challenges](#current-challenges)
+<!-- * [Differences between Sidecar Mode and Ambient Mode for Waypoint Proxy](#differences) -->
+* [Deciding the scope of your Waypoint Proxy](#Deciding-the-scope-of-your-Waypoint-Proxy)
+* [Functional Overview](#functional-overview)
+* [Deploying an Application](#deploying-an-application)
+* [Configuring Waypoint proxy](#configuring-waypoint-proxy)
+* [Monitoring the Waypoint Proxy & L7 Networking](#monitoring-the-waypoint-proxy--l7-networking)
+* [L7 Fault Injection](#l7-fault-injection)
+* [L7 Observability](#l7-observability)
+* [L7 Authorization Policy](#l7-authorization-policy)
+* [Co-existence of Ambient/ L7 with Side car proxies](#Co-existence-of-Ambient/-L7-with-Side-car-proxies)
+* [Control Traffic towards Waypoint Proxy](##control-traffic-towards-waypoint-proxy)
+* [Remove Waypoint proxy layer](#remove-waypoint-proxy-layer)
+
+## Deciding if you need A Waypoint proxy
+
+It's possible that the features offered by the secure overlay doesn’t meet your requirements. For instance, you need a rich Layer 7 authorization policy that sets up access based on a certain method and path. Alternatively you may like to conduct a canary test on the updated version of your service or introduce a new version without affecting current traffic. Or, you would like to receive metrics, HTTP access logs, and distributed tracing for some of your services. In order to accomplish these common cases, we'll go over how you can choose to enforce L7 processing with ambient mesh in this section.
+
+### Benefits of using the waypoint proxy and L7 networking features
+
+In summary, the waypoint proxy approach for the L7 processing layer offers the following three main advantages:
+
+- Security - Rich L7 authorization policy
+- Observability - HTTP metrics, access logs, and tracing
+- Traffic management - Dark launch, canary test
+
+The waypoint proxy and L7 networking features provide a number of benefits, including:
+
+- Improved performance and scalability: Waypoint proxies are designed to be lightweight and efficient, which can improve the performance and scalability of your microservices architecture.
+- Increased flexibility: The waypoint proxy allows you to implement a wide range of L7 networking features, such as HTTP load balancing, fault injection, and observability.
+- Simplified operations: By deploying a waypoint proxy, you can simplify the operation of your microservices architecture by reducing the number of components that need to be managed.
```

Comment:
I suggest remove this section as none of these benefits are convincing comparing with sidecars.